eng/scripts/Get-TypeSpec-Folders.ps1

```diff
@@ -13,6 +13,9 @@ $allChangedFiles = (Get-ChildItem -path ./specification tspconfig.yaml -Recurse)
 $allChangedFiles = $allChangedFiles -replace '\\', '/'
 
 if ([string]::IsNullOrEmpty($TargetBranch) -or [string]::IsNullOrEmpty($SourceBranch)) {
+  if ($TargetBranch -or $SourceBranch) {
```

Comment:
Why is extra if check necessary with the check on the line before? You only want to throw if one of them is not null/empty?